pkg/controller/command/presentproof/models.go

```diff
@@ -224,6 +224,8 @@ type SendProposePresentationArgs struct {
 	MyDID string `json:"my_did"`
 	// TheirDID receiver's did
 	TheirDID string `json:"their_did"`
+	// ConnectionID ID of connection between sender and receiver.
+	ConnectionID string `json:"connection_id"`
```

Comment:
if it is optional and substitute of `MyDID,TheirDID` then please mention in code comments.